ee-security/pom.xml

```diff
@@ -285,6 +264,30 @@
                 </plugins>
             </build>
         </profile>
+        <profile>
+            <id>integration-testing</id>
+            <build>
+                <plugins>
+                    <plugin>
+                        <groupId>org.apache.maven.plugins</groupId>
+                        <artifactId>maven-failsafe-plugin</artifactId>
+                        <configuration>
+                            <includes>
+                                <include>**/BasicRuntimeIT</include>
+                            </includes>
+                        </configuration>
+                        <executions>
+                            <execution>
+                                <goals>
+                                    <goal>integration-test</goal>
+                                    <goal>verify</goal>
+                                </goals>
+                            </execution>
+                        </executions>
+                    </plugin>
+                </plugins>
+            </build>
+        </profile>
         <profile>
```

Comment:
please remote this profile, no longer used or needed

Reply:
Hi @emmartins 
Thank you for the comments on the PR.
For this comment, are you referring to the arq-remote or the integration-testing profile?